src/routes/docs/tooling/command-line/buckets/+page.markdoc

```diff
@@ -0,0 +1,107 @@
+---
+layout: article
+title: Deploy buckets with CLI 2.0
```

Comment:
No need to name the CLI 2.0 anywhere in the docs. This detail is only relevant to marketing content.